{{cookiecutter.project_slug}}/.github/workflows/expo-pr.yml

```diff
@@ -50,3 +50,5 @@ jobs:
         env:
           EXPO_PUBLIC_BACKEND_SERVER_URL: {{ "${{ github.event.deployment_status.environment_url }}" }}
           EXPO_PUBLIC_ROLLBAR_ACCESS_TOKEN: ""
+          SENTRY_PROJECT_NAME: ""
+          SENTRY_DSN: ""
```

Comment:
These Sentry vars are needed, otherwise they will be blank in the `eas update` JS bundle and cause the app to crash on startup: #332